data/coalition/coalition jobs.txt

```diff
@@ -1322,3 +1322,122 @@ mission "Coalition: From Games 3"
 		payment
 		payment 25000
 		dialog phrase "coalition games dialog"
+
+
+mission "Heliarch Contain True"
+	job
+	repeat
+	name `Contain Criminals`
+	description `Fly to <destination> by <date> so that you and <bunks> other Heliarch agents can combat criminals on the site.`
+	deadline
+	passengers 16 58
+	to offer
+		random < 3
+		has "license: Heliarch"
+	source
+		government "Heliarch"
+	destination
+		distance 2 10
+		not attributes "coalition station"
+	on offer
+		require "Enforcer Confrontation Gear"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment
+		payment 70000
+		dialog `You and the other agents converge on the marked area and quickly move to cut off any exits. After a brief firefight the criminals surrender and a Heliarch ship swoops in to pick them up. You bid your Heliarch colleagues farewell and return to your ship where you are happy to see that <payment> has already been transferred to your account.`
+
+mission "Heliarch Contain Fail"
+	job
+	repeat
+	name `Contain Criminals`
+	description `Fly to <destination> by <date> so that you and <bunks> other Heliarch agents can combat criminals on the site.`
+	deadline
+	passengers 16 58
+	to offer
+		random < 15
+		has "license: Heliarch"
+	source
+		government "Heliarch"
+	destination
+		distance 2 10
+		not attributes "coalition station"
+	on offer
+		require "Enforcer Confrontation Gear"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		dialog `You and the other agents head to the marked area but when you arrive it becomes evident that whoever occupied the place has managed to escape. You bring the other agents to the spaceport as they lament the failed job.`
+
+mission "Heliarch Contain Fake"
+	job
+	repeat
+	name `Contain Criminals`
+	description `Fly to <destination> by <date> so that you and <bunks> other Heliarch agents can combat criminals on the site.`
+	deadline
+	passengers 16 58
+	to offer
+		random < 45
+		has "license: Heliarch"
+	source
+		government "Heliarch"
+	destination
+		distance 2 10
+		not attributes "coalition station"
+	on offer
+		require "Enforcer Confrontation Gear"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment
+		payment 55000
+		dialog `Your team pounces and catch the targets entirely by surprise. They offer no resistance, relinquishing their weapons immediately. You call for a Heliarch ship to pick them up and then bid the other agents farewell as you leave. An extra <payment> is already waiting in your account when you return to your ship.`
+
+
+mission "Heliarch Prisoner Transport"
+	job
+	repeat
+	name `Transport Prisoners`
+	description `Bring <bunks> criminals, who were arrested on <origin>, to <destination>. Payment is <payment>`
```

Comment:
This and the job below are missing a period at the end of the description, which other existing jobs have.